api/views/CloudRecordSummaryView.py

```diff
@@ -193,15 +193,17 @@ def _paginate_result(self, request, result):
         return serializer.data
 
     def _filter_cursor(self, cursor):
-        """Filter database results based on setting.RETURN_HEADERS."""
-        columns = cursor.description
-        results = []
-        for value in cursor.fetchall():
+        """Filter database results based on settings.RETURN_HEADERS."""
```

Comment:
I'll add "Allows for configuration of what summary fields the REST interface returns on GET requests."